moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -896,29 +896,49 @@ bool TrajectoryExecutionManager::validate(const TrajectoryExecutionContext &cont
     ROS_WARN_NAMED("traj_execution", "Failed to validate trajectory: couldn't receive full current joint state within 1s");
     return false;
   }
+  current_state->enforceBounds();
```

Comment:
I followed this call down to ``RevoluteJointModel::enforcePositionBounds()`` - so the comment should clarify that it isn't just enforcing bounds but also normalizing continuous joints